.github/workflows/build-and-test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Run Clippy
-        run: cargo clippy --all-targets --all-features
+        run: DISABLE_UNIFFI_EXPORT=1 cargo clippy --all-targets --all-features
```

Comment:
I think you can put it in `.cargo/config.toml`
so that other users won't need to set this env

Reply:
Good point! I updated at [64bca73](https://github.com/zkmopro/mopro/pull/321/commits/64bca730550711c6ecf90d221b4a9b30ae586e2f)